charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.0.1
+version: 1.0.2
```

Comment:
feature should bump to minor version.
```suggestion
version: 1.1.0
```

Reply:
Thanks for the catch! Change has been made :+1: